rag/export_training_data.py

```
@@ -1,185 +1,186 @@
+    text = re.sub(r"[\w\.-]+@[\w\.-]+\.[A-Za-z]{2,}", "<EMAIL>", text)
+
+    # URLs
+    text = re.sub(r"https?://\S+", "<URL>", text)
+
+    # Phone numbers (simple heuristic: international +81 or sequences of digits with separators)
+    text = re.sub(r"\+?\d[\d\-\s()]{6,}\d", "<PHONE>", text)
+
+    # 名前フィールド: 氏名: 山田太郎 -> 氏名: <PERSON>
+    text = re.sub(r"((?:氏名|名前|Name|Full ?Name)[:：])\s*[^\n,。;]+", r"\1 <PERSON>", text)
+
+    # 日本語の名前+敬称（例: 山田太郎さん -> <PERSON>さん）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)